pm/queue.go

```diff
@@ -80,7 +77,7 @@ func (q *ticketQueue) Redeemable() chan *redemption {
 
 // Length returns the current length of the queue
 func (q *ticketQueue) Length() (int, error) {
-	return q.store.WinningTicketCount(q.sender)
+	return q.store.WinningTicketCount(q.sender, new(big.Int).Sub(q.tm.LastInitializedRound(), big.NewInt(2)).Int64())
```

Comment:
Let's define 2 as a constant `ticketValidityPeriod` so we can just use that everywhere it is needed?

Reply:
fixed in f0ed1057